oap-server/server-alarm-plugin/src/test/resources/alarm-settings.yml

```diff
@@ -133,4 +133,10 @@ welinkHooks:
       # if you send to multi group at a time, separate group_ids with commas, e.g. "123xx","456xx"
       group_ids: "dummy_group_id"
       # make a name you like for the robot, it will display in group
-      robot_name: robot
\ No newline at end of file
+      robot_name: robot
+
+pagerDutyHooks:
```

Comment:
This file should be updated too, https://github.com/apache/skywalking/blob/f8203778efad77e1d1fb14f10a0000014b1a8517/oap-server/server-starter/src/main/resources/alarm-settings.yml

Reply:
@wu-sheng Thanks, fixed in https://github.com/apache/skywalking/pull/9247/commits/5cbb7388f94ed85261a2468f2940d2153e8e45cd :)